standard/unsafe-code.md

```diff
@@ -286,22 +287,23 @@ When one pointer type is converted to another, if the resulting pointer is not c
 
 > *Example*: Consider the following case in which a variable having one type is accessed via a pointer to a different type:
 >
-> <!-- Undefined$Example: {template:"standalone-console-without-using", name:"PointerConversions1", expectedWarnings:["CS8321"]} -->
+> <!-- Example: {template:"standalone-console-without-using", name:"PointerConversions1", expectedWarnings:["CS8321"]} -->
+> <!-- Note: the behavior of this example is undefined. -->
 > ```csharp
 > unsafe static void M()
 > {
 >     char c = 'A';
 >     char* pc = &c;
 >     void* pv = pc;
->     int* pi = (int*)pv;
->     int i = *pi; // undefined
->     *pi = 123456; // undefined
+>     int* pi = (int*)pv; // pretend a 16-bit char is a 32-bit int
+>     int i = *pi;        // read 32-bit int; undefined
+>     *pi = 123456;       // write 32-bit int; undefined
 > }
 > ```
 >
 > *end example*
 
-When a pointer type is converted to a pointer to `byte`, the result points to the lowest addressed `byte` of the variable. Successive increments of the result, up to the size of the variable, yield pointers to the remaining bytes of that variable.
+When a pointer type is converted to a pointer to `byte` (as with the cast to `void*` above), the result points to the lowest addressed `byte` of the variable. Successive increments of the result, up to the size of the variable, yield pointers to the remaining bytes of that variable.
```

Comment:
This appears to say that `void*` *is* a pointer to `byte`. (I'd also argue that we don't have a cast to `void*` in the code above - we're using an implicit conversion, but we're not using a cast.)

Worth discussing in next week's meeting?